src/main/kotlin/uk/gov/justice/digital/hmpps/prisonersearch/services/PrisonerEventListener.kt

```diff
@@ -34,6 +37,14 @@ class PrisonerEventListener(
       else -> log.warn("We received a message of event type {} which I really wasn't expecting", eventType)
     }
 
+    } catch (e: Exception) {
```

Comment:
Is there a test that could be updated?

Reply:
Not yet.  @matt-crowson-moj did this work initially, I'll raise another ticket for that part